app/controllers/katello/api/v2/products_controller.rb

```diff
@@ -47,7 +47,15 @@ def index
 
     def index_relation
       query = Product.readable.where(:organization_id => @organization.id)
-      query = query.where(:provider_id => @organization.anonymous_provider.id) if params[:custom]
+
+      if params[:custom]
+        if Foreman::Cast.to_bool(params[:custom])
+          query = query.where(:provider_id => @organization.anonymous_provider.id)
```

Comment:
while I am ok with this. I 'd prefer using scopes for custom and redhat in product model.

Reply:
good suggestion, I can look into it